### ISSUE TITLE
Validation loss never drops below 0.3

### ISSUE THREAD

**Issue by davidstrahm** [ultralytics/ultralytics#20014]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi
I have a question regarding the val/loss.

My dataset for classification consists of 10k images in 2 categories, split into 
train / val / test with a split of 0.7 / 0.2 / 0.1

I'm using ultralytics/ultralytics:8.3.19 (yolov11)
  
I get training accuracy of 1.0 after 39 epochs.

When validating with my test set using
`metrics = model.val(split='test', project=val_dir, plots=True, save_json=True)`

I also get a perfect score, also shown in the confusion matrix below.

However, the val/loss never drops below 0.31, no matter what I do. How can this be?
Note: I also get a perfect score when I use the val set for validation, because I thought maybe yolo uses the test set for training validation internally. So just to be sure, I ran validation on those files as well, with the same result.
  
Shouldn't the val/loss also drop to 0?  

```bash
epoch,time,train/loss,metrics/accuracy_top1,metrics/accuracy_top5,val/loss,lr/pg0,lr/pg1,lr/pg2
100,3856.34,0.0002,1,1,0.31328,0.000199,0.000199,0.000199
```
![Image](https://github.com/user-attachments/assets/153cd538-c998-4271-997c-cb4fa60f2b7f)

<img width="677" alt="Image" src="https://github.com/user-attachments/assets/8a2b0562-5a9d-4ab2-9448-9d11e40bb8b2" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @davidstrahm, thank you for your detailed report and for using Ultralytics 🚀!

We recommend checking out the [Ultralytics Docs](https://docs.ultralytics.com/) for guidance on training and validation, as well as exploring the [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) to ensure best practices.

If this is a 🐛 Bug Report, we kindly ask you to provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) so we can investigate further. This will help us identify any potential issues more effectively.

For your custom training ❓ Question, it would be helpful to share additional details, such as:
- A snippet of your training script or configuration
- A few dataset examples to understand the class distribution and image quality
- Training logs or other metrics that illustrate the behavior

If you're comparing metrics or losses, you may also want to verify that your dataset splits, augmentation settings, and hyperparameters align with your intended evaluation setup.

## Upgrade

Ensure you are using the latest version of the `ultralytics` package. You can upgrade it in your environment with:

```bash
pip install -U ultralytics
```

This ensures any recent fixes or improvements are applied. You can find all dependencies listed in the [requirements file](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) and verify compatibility with [Python>=3.8](https://www.python.org/) and [PyTorch>=1.8](https://pytorch.org/get-started/locally/).

## Environments

You can test your training or validation setup in any of these verified environments for a seamless experience:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Community and Support

Join the Ultralytics community to discuss this or similar topics:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time support
- [Discourse](https://community.ultralytics.com/) for detailed discussions
- [Subreddit](https://reddit.com/r/Ultralytics) for sharing insights and experiences

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI ensures YOLO tasks and modes perform as expected across macOS, Windows, and Ubuntu.

This is an automated response to guide you, but an Ultralytics engineer will review your issue soon 🔍. Thank you for your patience! 😊

**Comment by Y-T-G** [comment]
This was fixed in 8.3.47. You should upgrade to latest version.

https://github.com/ultralytics/ultralytics/pull/18085

**Comment by davidstrahm** [comment]
@Y-T-G thanks for the quick response! Works as expected now.
